app/models/user.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+module.exports = (sequelize, DataTypes) =>
+  sequelize.define(
+    'user',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        unique: true,
+        autoIncrement: true,
+        primaryKey: true
+      },
+      firstName: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      lastName: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      email: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        unique: true
+      },
+      password: {
+        type: DataTypes.STRING,
+        allowNull: false
+      }
+    },
+    {}
```

Comment:
We use snake_case in database (see [underscored](https://sequelize.org/v4/manual/tutorial/models-definition.html#configuration) option)

Reply:
Check this comment